.gitmodules

```diff
@@ -0,0 +1,28 @@
+[submodule "shared-modules"]
+	path = shared-modules
+	url = https://github.com/flathub/shared-modules.git
+	branch = master
+[submodule "rd-submodules/retroarch"]
```

Comment:
I'm not sure what's the point of these submodules?

Reply:
Building the emulators, otherwise they are not finding their resources.